azerothcore/azerothcore-wotlk
# fix(Core/Spell): Holy Priest Tier 5 Greater Heal Refund proc

There is a PR https://github.com/azerothcore/azerothcore-wotlk/pull/1765 open but isn't needed.

###### ISSUES ADDRESSED:
Holy Priest Tier 5 (2 pieces) bonus  http://wotlk.cavernoftime.com/spell=37594 procs any time greater heal is cast if you have at least two pieces off this tier equiped.

Closes 

##### TESTS PERFORMED: 
Build without errors and tested in-game. Win10  server.

##### HOW TO TEST THE CHANGES:
. create a priest lvl >=70
. add and equip at least 2p from the list (30153,30152,30151,30154,30150) 
. cast greater heal on any healable target including yourself (on effective heals that brings your target to full health you will gain 100 mana as expected)

##### Target branch(es):

Master
